Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -538,6 +542,60 @@ void MemoryWidget::OnSetValue()
   Update();
 }
 
+void MemoryWidget::OnSetValueFromFile()
+{
+  if (!Core::IsRunning())
+    return;
+
+  bool good_address, good_offset;
+  // Returns 0 if conversion fails
+  u32 addr = m_search_address->text().toUInt(&good_address, 16);
```

Comment:
This part LGTM, it matches the current offset behaviour.

Note to myself if this is merged before https://github.com/dolphin-emu/dolphin/pull/10079, I'll need to update this as well.

Reply:
I merged #10079 first, could you update this to use the new `GetTargetAddress()` function?